src/mx/NotificationSenderInterface.java

```diff
@@ -0,0 +1,9 @@
+package mx;
+
+/**
+ * Created by masterix on 28.05.2017.
+ */
+public interface NotificationSenderInterface {
```

Comment:
`NotificationSender` would be better - no need to mark the fact it's an interface